test/inventory/hosts

```diff
@@ -0,0 +1,4 @@
+[fixtures-group]
+fixtures ansible_connection=local
+[tests-group]
```

Comment:
you shouldn't need groups here

Reply:
You actually shouldn't need an inventory file if you pass everything inline in the ansible-playbook command: https://github.com/theforeman/foreman-ansible-modules/compare/master...akofink:vcr#diff-33830cc555322fe03af4593b369e4359R11

Reply:
I like the inventory as a file (not too strong feelings, though).